package.json

```diff
@@ -57,7 +57,7 @@
     "lint": "eslint -c .eslintrc.json src test",
     "version": "echo $npm_package_version",
     "start": "node bin/load.js",
-    "start:recent-trials": "node bin/load.js api clinicaltrialsgov",
+    "start:recent-trials": "node bin/load.js clinicaltrialsgov --days 7",
```

Comment:
7 days seems a bit short. @elewis2, what about 14 days so we are still good if a cron fail?